src/Parsers/MySQL/ASTCreateQuery.cpp

```diff
@@ -63,19 +63,19 @@ bool ParserCreateQuery::parseImpl(Pos & pos, ASTPtr & node, Expected & expected)
     if (ParserKeyword("IF NOT EXISTS").ignore(pos, expected))
         if_not_exists = true;
 
-    if (!ParserCompoundIdentifier(false).parse(pos, table, expected))
+    if (!ParserCompoundIdentifier(true).parse(pos, table, expected))
         return false;
 
     if (ParserKeyword("LIKE").ignore(pos, expected))
     {
-        if (!ParserCompoundIdentifier(false).parse(pos, like_table, expected))
+        if (!ParserCompoundIdentifier(true).parse(pos, like_table, expected))
```

Comment:
Why we change table_name_with_optional_uuid_ ctor parameter to true?

Reply:
Because `ParserCompoundIdentifier(true)` stands for now for `ParseTableIdentifier` - it returns `ASTTableIdentifier` and parses it accordingly (i.e. with UUID).